src/componentLibrary/CoinPair/CoinPair.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  Story,
+  Meta
+} from '@storybook/react';
+
+import { CoinPair, CoinPairProps } from './';
+
+const Template: Story<CoinPairProps> = args => <CoinPair {...args} />;
+
+const Small = Template.bind({});
+Small.args = {
+  coinOne: 'btc',
+  coinTwo: 'ksm',
+  size: 'small'
+};
+
+const Large = Template.bind({});
+Large.args = {
+  coinOne: 'btc',
+  coinTwo: 'ksm',
+  size: 'large'
```

Comment:
Hi Dan, there is a reason for this :) I prefer that the components can be used without needing to import a type. By using string literals you can do something like this:

`<CoinPair coinOne='btc' coinTwo='ksm' />`

The editor is still aware of the types so you get autocomplete and type safety for the property values.

If you use an enum you need to import the type:

```
import { Coins } from 'componentLibrary/types;

<CoinPair coinOne={Coins.Btc} coinTwo={Coins.Ksm} />
```

For me, this second way of doing things adds a bit of unnecessary complexity without adding anything (since we already have type safety with the string literals.